backend/src/proto/gradido.proto.js

```diff
@@ -0,0 +1,4805 @@
+                        object.versionNumber = options.longs === String ? "0" : 0;
+                }
+                if (message.memo != null && message.hasOwnProperty("memo"))
+                    object.memo = message.memo;
+                if (message.created != null && message.hasOwnProperty("created"))
+                    object.created = $root.proto.gradido.TimestampSeconds.toObject(message.created, options);
+                if (message.versionNumber != null && message.hasOwnProperty("versionNumber"))
+                    if (typeof message.versionNumber === "number")
+                        object.versionNumber = options.longs === String ? String(message.versionNumber) : message.versionNumber;
+                    else
+                        object.versionNumber = options.longs === String ? $util.Long.prototype.toString.call(message.versionNumber) : options.longs === Number ? new $util.LongBits(message.versionNumber.low >>> 0, message.versionNumber.high >>> 0).toNumber(true) : message.versionNumber;
+                if (message.transfer != null && message.hasOwnProperty("transfer")) {
+                    object.transfer = $root.proto.gradido.GradidoTransfer.toObject(message.transfer, options);
+                    if (options.oneofs)
+                        object.data = "transfer";
+                }
+                if (message.creation != null && message.hasOwnProperty("creation")) {
+                    object.creation = $root.proto.gradido.GradidoCreation.toObject(message.creation, options);
+                    if (options.oneofs)
+                        object.data = "creation";
+                }
+                if (message.groupFriendsUpdate != null && message.hasOwnProperty("groupFriendsUpdate")) {
+                    object.groupFriendsUpdate = $root.proto.gradido.GroupFriendsUpdate.toObject(message.groupFriendsUpdate, options);
+                    if (options.oneofs)
+                        object.data = "groupFriendsUpdate";
+                }
+                if (message.groupMemberUpdate != null && message.hasOwnProperty("groupMemberUpdate")) {
+                    object.groupMemberUpdate = $root.proto.gradido.GroupMemberUpdate.toObject(message.groupMemberUpdate, options);
+                    if (options.oneofs)
+                        object.data = "groupMemberUpdate";
+                }
+                return object;
+            };
+
+            /**
+             * Converts this TransactionBody to JSON.
+             * @function toJSON
+             * @memberof proto.gradido.TransactionBody
+             * @instance
+             * @returns {Object.<string,*>} JSON object
+             */
+            TransactionBody.prototype.toJSON = function toJSON() {
+                return this.constructor.toObject(this, $protobuf.util.toJSONOptions);
+            };
+
+            return TransactionBody;
+        })();
+
+        return gradido;
+    })();
+
+    return proto;
+})();
+
+module.exports = $root;
```

Comment:
WTF

Reply:
This whole file is generated by the proto library. I checked in the generated file since I do not want to require everyone to build this stuff everytime - just have it statically linked till we remove it again.